ClinGen/clincoded
# Connects to #974. Connects to #836. Status behavior and slight changes to Orphanet term rendering

#### Connects to #974:

* Changed calculated status for interpretations: 'provisional' if there is provisional classification; 'in progress' in all other cases (0 or more interpretations saved)
* Action of adding interpretation now shows up on dashboard history
* Variant w/ no evaluations now show up on variant list under 'My Variants' list on dashboard

Testing:

1. Go into VCI, and add a Variant
2. Add an interpretation, and confirm that a history item and a My Variant entry exists on the dashboard. Confirm that the interpretation has a status of 'In Progress'
3. Add an evaluation and confirm that the interpretation is still 'In Progress'
4. Add a provisional classification and confirm that the interpretation status is now 'Provisional'

#### Connects to #836:

* Note: Was connected to #839 as well
* Note: Both tickets were made somewhat outdated by changes to the interpretation list format in other tickets
* Puts ORPHA### in parentheses instead of after a comma
